Applications/Terminal/main.cpp

```diff
@@ -124,15 +125,35 @@ int main(int argc, char** argv)
     slider->set_range(0, 100);
     slider->set_value(100);
 
+    auto* beep_choice_window = new GWindow;
+    beep_choice_window->set_title("Terminal beep settings");
```

Comment:
We should eventually merge this with the opacity settings and whatever else we can think of into a single settings window.
It's not crucial that we do it now though :)